lib/commands/functions.js

```diff
@@ -405,6 +405,7 @@ const functionsCreateDeployment = async ({ functionId, code, activate, entrypoin
     /* @param {string} entrypoint */
     /* @param {string} commands */
 
+    try{
```

Comment:
1. Let's not have such a big try/catch
2. This repo is read-only and the change should be done with the [sdk-generator](https://github.com/appwrite/sdk-generator) repo.